src/components/VisualSkeletalAnimation.hpp

```diff
@@ -0,0 +1,270 @@
+/** \file
```

Comment:
Default doxygen comment leftover?

Reply:
This is needed for the file to be recognized by Doxygen at all. The documentation can be found at the Class then.